src/file/stat.cr

```diff
@@ -100,8 +100,8 @@ class File
     end
 
     def inspect(io)
-      io << "#<File::Stat"
-      io << " dev=0x"
+      io << "File::Stat("
+      io << "dev=0x"
```

Comment:
Can be merged to `io << "File::Stat(dev=0x"`

Reply:
Nope, that just looks weird, especially since there's many lines below having such format.

Reply:
Why weird? Aesthetics? I don't see why these strings should not be combined.